### PR TITLE
Extract function `guitypes`

### PR DESCRIPTION
**Pull request by egeerardyn**
This removes the only FIXME from `figure2dot`.

Note: only executed mentally, so wait for Jenkins and Travis to be sure.

### CODE REVIEW

File: src/private/guitypes.m [hunk 1]
```diff
@@ -0,0 +1,13 @@
+function types = guitypes()
+% GUITYPES returns a cell array of MATLAB/Octave GUI object types
+%
+% Syntax
+%   types = guitypes()
+%
+% These types are not ignored by matlab2tikz and figure2dot.
```
Comment (PeterPablo) [inline src/private/guitypes.m]: `not`?

File: src/matlab2tikz.m [hunk 2]
```diff
@@ -681,9 +681,8 @@ function addCustomCode(fid, before, code, after)
             case 'histogram'
                 [m2t, str] = drawHistogram(m2t, child);
 
-            case {'uitoolbar', 'uimenu', 'uicontextmenu', 'uitoggletool',...
-                    'uitogglesplittool', 'uipushtool', 'hgjavacomponent'}
-                % don't to anything for these handles and its children
+            case guitypes
```
Comment (PeterPablo) [inline src/matlab2tikz.m]: `guitypes()` to emphasize that this is a function?

File: src/figure2dot.m [hunk 3]
```diff
@@ -106,8 +106,7 @@ function plot_children(fh, h, parent_node)
 function bool = shouldSkip(h)
     %  returns TRUE for objects that can be skipped
     objType = get(h, 'Type');
-    bool = ismember(lower(objType), {'uimenu', 'uitoolbar', 'uicontextmenu'});
-    %FIXME: maybe integrate this in matlab2tikz?
+    bool = ismember(lower(objType), guitypes);
```
Comment (PeterPablo) [inline src/figure2dot.m]: `guitypes()` to emphasize that this is a function?

File: src/matlab2tikz.m (hunk 2 above)
Comment (egeerardyn) [inline src/matlab2tikz.m]: :+1:

File: src/figure2dot.m (hunk 3 above)
Comment (egeerardyn) [inline src/figure2dot.m]: Please see the other one :stuck_out_tongue:

File: src/private/guitypes.m (hunk 1 above)
Comment (PeterPablo) [inline src/private/guitypes.m]: What about this comment? The way I read the code, those types *are* ignored by m2t and f2d.

Comment (egeerardyn) [inline src/private/guitypes.m]: Hmm, you are right `not` should NOT be there.

Comment (PeterPablo) [inline src/private/guitypes.m]: yeah... I should have written ~~`not`~~.

Comment (egeerardyn) [inline src/private/guitypes.m]: No, I overlooked this comment altogether. Thanks for making me aware!

**Pull request merged**